mikeio/dfs/_dfs.py

```diff
@@ -391,10 +391,16 @@ def read(
         elif self._ndim == 2:
             shape = (nt, self.ny, self.nx)  # type: ignore
         else:
+            # TODO this is not used, since Dfs3 has a separate .read method
             shape = (nt, self.nz, self.ny, self.nx)  # type: ignore
 
+        spdims = self.geometry.default_dims
+        dims = ["time"]
```

Comment:
It seems a little confusing to me to first always prepend dims with "time" and then afterwards potentially removing it in line 403 🤔 But it looks like it works and you have probably considered alternative ways.

Reply:
There is room for improvement...